### ISSUE TITLE
ESC fails switch to evil mode?

### ISSUE THREAD

**Issue by fuzihaofzh** [syl20bnr/spacemacs#6399]
I want to switch into eval mode, and I press ESC, it shows:

```
 ➊  * 0 *scratch*  Text  Ⓔh Ⓚ                                            utf-8 | 1: 0  [⇪]  All
Wrong number of arguments: #[(&optional continue) "^H\205^[^@   ?\205^[^@Ä \210
\204^X^@^K¢\203^X^@Å \210Æ\211^P\207" [evil-undo-list-pointer evil-in-single-undo continue buffer-undo\
-list evil-refresh-undo-step undo-boundary nil] 2 ("/global-hadoop/home/me/.emacs.d/elpa/evil-201\
60619.2253/evil-common.elc" . 102231)], 2
```

**Comment by maplewizard** [comment]
@TheBB @d12frosted How can I provide more information abuot this issue?

**Comment by syl20bnr** [comment]
Update your Spacemacs, it is fixed in 0.105.21.

**Comment by syl20bnr** [comment]
> How can I provide more information abuot this issue?

You can press `SPC h I` (capital i) in Spacemacs to open a template prefilled with precious info for us to diagnose the issue.

But I know this bug and from what I can conclude is that:
- you are not on 0.105.21
- you are using Hybrid style.
